### PR TITLE
Added big rocks to the table

### PR DESCRIPTION
**Pull request by linaori**
Because why not 😄

Size wise it's the same as sand-rock-big, but in the same style as rock-huge.

### CODE REVIEW

File: map_gen/Diggy/Feature/DiggyCaveCollapse.lua [hunk 1]
```diff
@@ -112,15 +113,15 @@ local function create_collapse_template(positions, surface)
 
         if do_insert then
             entity_count = entity_count + 1
-            entities[entity_count] = {position = {x = x, y = y}, name = 'sand-rock-big'}
+            entities[entity_count] = {position = position, name = collapse_rocks[random(1, collapse_rocks_size)]}
```
Comment (mheguy) [inline map_gen/Diggy/Feature/DiggyCaveCollapse.lua]: If you wanna be 😎 cool you could just use `random(collapse_rock_size)`. The lower limit of 1 is implies.

File: map_gen/Diggy/Feature/DiggyHole.lua [hunk 2]
```diff
@@ -127,7 +130,7 @@ function DiggyHole.register(config)
     Event.add(defines.events.on_entity_died, function (event)
         local entity = event.entity
         local name = entity.name
-        if name ~= 'sand-rock-big' and name ~= 'rock-huge' then
+        if name ~= 'sand-rock-big' and name ~= 'rock-huge' and name ~= 'rock-big' then
```
Comment (mheguy) [inline map_gen/Diggy/Feature/DiggyHole.lua]: This exact rock check and return pops up at least 4 times in this file, would like to see it split into a function.

**Review by mheguy — APPROVED** [review]
Ran into no issues during testing and no issues popped out in looking at the code. Would like to see the function created for that rock check, but it's not something that blocks this merge IMO.

File: map_gen/Diggy/Feature/DiggyCaveCollapse.lua (hunk 1 above)
Comment (linaori) [reply map_gen/Diggy/Feature/DiggyCaveCollapse.lua]: I knew that! 😅

File: map_gen/Diggy/Feature/DiggyHole.lua (hunk 2 above)
Comment (linaori) [reply map_gen/Diggy/Feature/DiggyHole.lua]: I think that's indeed a nice thing to have about now. I'll fix this once I get back home later today

**Comment by linaori** [comment]
Fixed the random and moved the definition of the diggy rocks table + check to the Template lua file

**Comment by mheguy** [comment]
Looks good to me, merging with my thanks.

**Pull request merged**